integration/debug_test.go

```diff
@@ -40,12 +40,13 @@ func TestDebug(t *testing.T) {
 		{
 			description: "kubectl",
 			dir:         "testdata/debug",
+			args:        []string{"--cache-artifacts=false"},
 			deployments: []string{"jib"},
 			pods:        []string{"nodejs", "npm", "python3"},
 		},
 		{
 			description: "kustomize",
-			args:        []string{"--profile", "kustomize"},
```

Comment:
is there a reason why we turn off caching for these two tests?

Reply:
never mind. just saw your commit message "Add --cache-artifacts=false for integrationt tests where we check builds"

Reply:
but these are debug test? do we still need to disable caching?